.github/workflows/sync-examples.yml

```diff
@@ -0,0 +1,29 @@
+name: '[CI] Sync Examples'
+
+on:
+    push: # Only run on merges into master that modify files under pyscriptjs/
```

Comment:
```suggestion
    push: # Only run on merges into main that modify files under pyscriptjs/
```

Reply:
Thanks - force of habit from team edition.